app/jobs/discourse_activity_pub_log_rotate.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Jobs
+  class DiscourseActivityPubLogRotate < ::Jobs::Scheduled
+    every 1.day
+
+    def execute(args)
+      return if !SiteSetting.activity_pub_enabled
```

Comment:
Should we run this cleanup job regardless of whether the plugin is enabled or not?